desktop/apps/consign/routes.js

```diff
@@ -20,8 +20,8 @@ export const landingPage = async (req, res, next) => {
     inDemand.id = in_demand.set.id
 
     await inDemand.fetch({ cache: true })
-    const { ordered_set: recentlySold } = await metaphysics({ query: RecentlySoldQuery(recently_sold.set.id) })
-    const { sales } = await metaphysics({ query: SalesQuery() })
+    const { ordered_set: recentlySold } = await metaphysics({ query: RecentlySoldQuery(recently_sold.set.id), req })
+    const { sales } = await metaphysics({ query: SalesQuery(), req })
```

Comment:
I wonder if we could DRY this up somehow without having to explicitly pass `req` every time. Given that this `metaphysics` library is meant to be isomorphic—I don't see how that's possible. We could provide a helper middleware like `req.metaphysics`, but that doesn't feel any better. 🤔 

👍  to this then... just wish we didn't need to have the cognitive overhead of remembering to pass `req` in every time.